doc/changes/latest.rst

```diff
@@ -18,3 +18,5 @@ Enhancements
 
 Changes
 -------
+
+:bdg-dark:`Code` Move ``nilearn.plotting.img_plotting`` under ``nilearn.plotting.image`` (:gh:`5481` by `Hande Gözükan`_).
```

Comment:
@Remi-Gau I remember you telling me sometime ago that we might name the package as `nilearn.plotting.volume` rather then `nilearn.plotting.image` as surface images are also images. Shall I change that to volume?

Reply:
No strong opinion on that. This is a bit more consistent maybe. We should look a bit more systematically at the naming used so far.

Reply:
yes that makes sense to me as long as we have a clear division between the 2